tests/helpers/drag.js

```diff
@@ -0,0 +1,82 @@
+import Ember from 'ember';
+
+/**
+  Drags elements by an offset specified in pixels.
+
+  Examples
+
+      drag(
+        'mouse',
+        '.some-list li[data-item=uno]',
+        { y: 50 }
+      );
+
+  @method drag
+  @param {'mouse'|'touch'} [mode]
+    event mode
+  @param {String} [itemSelector]
+    selector for the element to drag
+  @param {Function} [offsetFn]
+    function returning the offset by which to drag
+  @param {Object} [callbacks]
+    callbacks that are fired at the different stages of the interaction
+  @return {Promise}
+*/
+
+export function drag(_app, mode, itemSelector, offsetFn, callbacks = {}) {
+  let start, move, end;
+
+  if (mode === 'mouse') {
+    start = 'mousedown';
+    move = 'mousemove';
+    end = 'mouseup';
+  } else if (mode === 'touch') {
+    start = 'mousedown';
+    move = 'mousemove';
+    end = 'mouseup';
```

Comment:
I just realised that I messed up my reorder helper and never used the touchevent names!